tools/individual-packages-gen/gen-phase1.ts

```diff
@@ -0,0 +1,209 @@
+import * as path from 'path';
+import * as awsCdkMigration from 'aws-cdk-migration';
+import * as fs from 'fs-extra';
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const lerna_project = require('@lerna/project');
+
+copyFilesRemovingDependencies();
+
+function copyFilesRemovingDependencies(): void {
+  // there is a lerna.json in the individual-packages directory, where this script executes
+  const project = new lerna_project.Project(__dirname);
+  const packages = project.getPackagesSync();
+  const alphaPackages = getAlphaPackages(packages);
+  const jsiiRosettaVersion = project.manifest.devDependencies['jsii-rosetta'];
+  for (const pkg of packages) {
+    if (!packageIsAlpha(pkg)) {
+      continue;
+    }
+
+    const srcDir = pkg.location;
+    const packageUnscopedName = `${pkg.name.substring('@aws-cdk/'.length)}`;
+    const destDir = path.join('.', packageUnscopedName);
+    fs.mkdirpSync(destDir);
+
+    copyOrTransformFiles(pkg, srcDir, destDir);
+  }
+
+  function copyOrTransformFiles(pkg: any, srcDir: string, destDir: string): void {
+    const sourceFiles = fs.readdirSync(srcDir);
+    for (const sourceFileName of sourceFiles) {
+      if (shouldIgnoreFile(sourceFileName)) {
+        continue;
+      }
+
+      const source = path.join(srcDir, sourceFileName);
+      const destination = path.join(destDir, sourceFileName);
+
+      let fileProcessed = false;
+      if (sourceFileName === 'package.json') {
+        const srcPackageJson = fs.readJsonSync(source);
+        if (srcPackageJson.name === pkg.name) {
+          const pkgUnscopedName = `${pkg.name.substring('@aws-cdk/'.length)}`;
+
+          srcPackageJson.name = `@aws-cdk-lib-alpha/${pkgUnscopedName}`;
+          srcPackageJson.repository.directory = `packages/individual-packages/${pkgUnscopedName}`;
+
+          // JSII targets
+          const jsiiTargets = srcPackageJson.jsii.targets;
+          jsiiTargets.dotnet.namespace = jsiiTargets.dotnet.namespace.replace(
+            /^Amazon\.CDK\./, 'Amazon.CDK.Alpha.');
+          jsiiTargets.java.package = jsiiTargets.java.package.replace(
+            /^software\.amazon\.awscdk\./, 'software.amazon.awscdk.alpha.');
+          jsiiTargets.java.maven.artifactId = jsiiTargets.java.maven.artifactId.startsWith('cdk-')
+            ? jsiiTargets.java.maven.artifactId.replace(/cdk-/, 'cdk-alpha.')
+            : 'cdk-alpha.aws-' + jsiiTargets.java.maven.artifactId;
+          jsiiTargets.python.distName = jsiiTargets.python.distName.replace(
+            /^aws-cdk\./, 'aws-cdk.alpha.');
+          jsiiTargets.python.module = jsiiTargets.python.module.replace(
+            /^aws_cdk\./, 'aws_cdk.alpha.');
```

Comment:
Make these changes directly in the package.json source file on the v2 branch, enforced by pkglint.

Reply:
Why?

This will mean we will have to revert these changes again the moment the package graduates from "alpha" to "stable". I don't see a reason to add this additional churn to these packages.

Reply:
To be clear, if your answer is "pkglint will do this for us automatically", it will not (it loses the correct casing, and periods in a few places, like the C# namespace).

Reply:
I've made this change, but in my opinion it's a huge mistake.